build/gulp/scss/replacements.js

```diff
@@ -0,0 +1,248 @@
+'use strict';
```

Comment:
This file is the result of the wrong merge. It was renamed here https://github.com/DevExpress/DevExtreme/commit/2c04d26b99956ec05639aad340514f325ce39615#diff-d52f1d98780d844cec02bf2731ebb66c